code/modules/mod/mod_core.dm

```diff
@@ -355,3 +355,70 @@
 	add_charge(uses_needed * charge_given)
 	balloon_alert(user, "core refueled")
 	return TRUE
+
+/obj/item/mod/core/plasma/lavaland
+	name = "MOD plasma flower core"
+	icon_state = "mod-core-plasma-flower"
+	desc = "A strange flower from the desolate wastes of lavaland. It pulses with a strange purple glow.  \
+		The wires coming out of it could be hooked into a MODsuit."
+	light_system = MOVABLE_LIGHT
+	light_color = "#cc00cc"
+	light_range = 2
+
+	var/mob/living/mob = /mob/living/basic/butterfly/lavaland/temporary
```

Comment:
These also need documenting comments, and this should probably have a more specific name.